examples/multiphysics/rising-bubble/rising-bubble-comparison.py

```diff
@@ -0,0 +1,290 @@
+# Data TP2D in Hysing, S., Turek, S., Kuzmin, D., Parolini, N., Burman, E., Ganesan, S., & Tobiska, L. (2009). Quantitative benchmark computations of two‐dimensional bubble dynamics. International Journal for Numerical Methods in Fluids, 60(11), 1259-1288.
+x_ref_TP2D = [0.002, 0.246, 0.493, 0.736, 0.988, 1.233, 1.485, 1.729, 1.975, 2.226, 2.471, 2.717, 3.000]
+y_ref_TP2D = [0.501, 0.517, 0.563, 0.623, 0.685, 0.741, 0.796, 0.853, 0.909, 0.968, 1.024, 1.077, 1.138]
+x_vel_TP2D = [0.000, 0.245, 0.498, 0.748, 0.992, 1.243, 1.496, 1.748, 1.994, 2.245, 2.495, 2.746, 2.999]
+y_vel_TP2D = [0.001, 0.139, 0.230, 0.253, 0.242, 0.229, 0.226, 0.232, 0.243, 0.232, 0.220, 0.217, 0.217]
+df_contour_TP2D = pd.read_csv("reference_contour/case2_contour_TP2D.csv", header=0)
+
+# Data FreeLIFE in Hysing, S., Turek, S., Kuzmin, D., Parolini, N., Burman, E., Ganesan, S., & Tobiska, L. (2009). Quantitative benchmark computations of two‐dimensional bubble dynamics. International Journal for Numerical Methods in Fluids, 60(11), 1259-1288.
+x_ref_FreeLIFE = [0.005, 0.246, 0.498, 0.750, 1.000, 1.246, 1.499, 1.750, 2.000, 2.247, 2.496, 2.748, 3.000]
+y_ref_FreeLIFE = [0.501, 0.517, 0.563, 0.625, 0.685, 0.742, 0.797, 0.853, 0.912, 0.969, 1.023, 1.073, 1.125]
+x_vel_FreeLIFE = [0.000, 0.244, 0.496, 0.748, 0.998, 1.245, 1.496, 1.747, 1.999, 2.243, 2.496, 2.747, 2.999]
+y_vel_FreeLIFE = [0.001, 0.138, 0.229, 0.252, 0.239, 0.227, 0.224, 0.232, 0.244, 0.225, 0.212, 0.208, 0.205]
+df_contour_FreeLIFE = pd.read_csv("reference_contour/case2_contour_FreeLIFE.csv", header=0)
+
+# Data MooNMD in Hysing, S., Turek, S., Kuzmin, D., Parolini, N., Burman, E., Ganesan, S., & Tobiska, L. (2009). Quantitative benchmark computations of two‐dimensional bubble dynamics. International Journal for Numerical Methods in Fluids, 60(11), 1259-1288.
+x_ref_MooNMD = [0.002, 0.246, 0.493, 0.736, 0.988, 1.233, 1.485, 1.729, 1.975, 2.226, 2.471, 2.717, 3.000]
+y_ref_MooNMD = [0.501, 0.517, 0.563, 0.623, 0.685, 0.741, 0.796, 0.853, 0.909, 0.968, 1.024, 1.077, 1.138]
+x_vel_MooNMD = [0.246, 0.487, 0.735, 0.986, 1.231, 1.482, 1.728, 1.973, 2.226, 2.471, 2.715, 0.002, 2.999]
+y_vel_MooNMD = [0.137, 0.226, 0.250, 0.239, 0.226, 0.222, 0.228, 0.238, 0.231, 0.218, 0.215, 0.001, 0.213]
+df_contour_MooNMD = pd.read_csv("reference_contour/case2_contour_MooNMD.csv", header=0)
+
+
+# Load position and velocity of the barycenter
+t_sharp,x_sharp,y_sharp,vx_sharp,vy_sharp=np.loadtxt(filename_barycenter_sharp,skiprows=1,unpack=True)
+t_geo,x_geo,y_geo,vx_geo,vy_geo=np.loadtxt(filename_barycenter_geo,skiprows=1,unpack=True)
+t_alge,x_alge,y_alge,vx_alge,vy_alge=np.loadtxt(filename_barycenter_alge,skiprows=1,unpack=True)
+
+
+# Plot the position of the barycenter
+fig0 = plt.figure()
+ax0 = fig0.add_subplot(111)
+ax0.plot(t_sharp, y_sharp, '-', lw=2, label="Projection")
+ax0.plot(t_geo, y_geo, '-', lw=2, label="Geometric")
+ax0.plot(t_alge, y_alge, '-', lw=2, label="PDE-based")
+
```

Comment:
I guess what you could do to make this script general is not make the arguments mandatory above but make them optional and then decide what to plot depending on which one is provided?